source/localizable/smart_cart/orders_api.html.md.erb

```diff
@@ -91,14 +111,37 @@ The available rejection reason IDs are provided in `reject_options` when retriev
 
 <%= render_recording :merchants_ecommerce_orders_reject_other %>
 
+#### Example
+<pre class="terminal">
+curl -X POST https://<%= settings.api_domain %>/merchants/ecommerce/orders/191128-1234567/reject \
+  -H 'Accept: application/vnd.<%= settings.site_name %>+json; version=3.0' \
+  -H 'Content-Type: application/json; charset=utf-8' \
+  -H 'Authorization: Bearer your_access_token_here' \
+  -d '{ "rejection_reason_other": "Our store is closed for personal reasons" }'
+</pre>
+
 ## Error handling
 
 In case of errors, the response will have a representative HTTP status code
 in the `4xx` or `5xx` range. The body would contain an array of `errors`.
 
-For example, if you try to reject an order that is already rejected, you will
+For example, if you try to accept an order that is already accepted, you will
 get a `422` HTTP status code (Unprocessable Entity), with the following body:
 
+<%= render_code '{
+  "errors": [
+      {
+          "code": "order_status",
+          "messages": [
+              "Order already accepted."
+          ]
+      }
+  ]
+}', 'javascript' %>
+
+And if you try to reject an order that is already rejected, you will
+also get a `422` HTTP status code (Unprocessable Entity), with the following body:
+
 <%= render_code '{
```

Comment:
I'd suggest adding a request example with invalid request headers, i.e. an example missing the auth token header or having an invalid one. We don't have to explicitly refer to a valid access token value, just a descriptive name and the expected response in these cases (`401 Unauthorized`):

```
{"errors":[{"code":"Unauthorized","messages":["unauthorized"]}]}
```
We could also simply state that the above is expected in case the aforementioned header is missing or the token is invalid.

WDYT? cc @nikostsaganos

Reply:
The section for general error handling is still there though and it includes the 401 case. Does it make sense to include it here as well?

Reply:
Ignore me, we already have a `More on error handling` link, forgot this :)